.circleci/config.yml

```diff
@@ -238,40 +283,45 @@ jobs:
             pip3.8 install conbench
             pip3.8 install git+https://github.com/conbench/benchalerts.git@0.3.1
             pip3.8 install scripts/veloxbench
+      - run-benchmarks
+      # TODO: Disabling conbench github notifications for now until results are more stable.
+      #- run:
+      #  name: "Analyze Benchmark Regressions and Post to Github"
+      #    command: |
+      #      export GITHUB_APP_PRIVATE_KEY=$(echo $GITHUB_APP_PRIVATE_KEY_ENCODED | base64 -d)
+      #      python3.8 scripts/benchmark-github-status.py --z-score-threshold 50
+      - post-steps
+
+  linux-benchmarks-basic-dedicated:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: 2xlarge
+    environment:
+      CONBENCH_URL: "https://velox-conbench.voltrondata.run/"
+      CONBENCH_HOST: "CircleCI-runner-dedicated"
```

Comment:
Nice! We might consider just calling this `dedicated` or `aws-dedicated` to make it stand out a bit more from the Circle-CI hosted runner, but this is totally fine to use too